docs/developer/operator_overview.md

```diff
@@ -0,0 +1,673 @@
+
+When the operator detects this prefix, it creates separate property files for each targeted ordinal within the secret, ensuring each broker gets its specific configuration.
+
+```yaml
+spec:
+  brokerProperties:
+    # This applies to all brokers
+    - "globalMaxSize=512m"
+    # This applies ONLY to the broker pod with ordinal 0 (e.g., ex-aao-ss-0)
+    - "broker-0.management-address=prod.management.0"
+    # This applies ONLY to the broker pod with ordinal 1 (e.g., ex-aao-ss-1)
+    - "broker-1.management-address=prod.management.1"
+```
+
+#### Providing Properties from an External Secret
+
+To better organize your properties or to source them from a different management system, you can provide them in a separate Kubernetes `Secret` and mount it into the broker.
+
+If a secret mounted via `extraMounts` has a name ending with the suffix `-bp` (for **b**roker **p**roperties), the operator will automatically treat it as a source of broker properties and include it in the broker's configuration path.
+
+Properties are applied in order: first the properties from the CR's `brokerProperties` field, then the properties from each `-bp` secret, loaded in alphabetical order by secret name.
+
+**1. Create a secret with a `-bp` suffix:**
+
+The secret can contain multiple keys, where each key represents a file. The content of each file should be in Java properties format.
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: my-extra-config-bp
+stringData:
+  global-mem.properties: |
+    globalMaxSize=1G
+  tuning.properties: |
+    journal-buffer-timeout=10000
+```
+
+**2. Reference the secret in the `ActiveMQArtemis` CR:**
+
+```yaml
+apiVersion: broker.amq.io/v1beta1
+kind: ActiveMQArtemis
+metadata:
+  name: ex-aao
+spec:
+  deploymentPlan:
+    extraMounts:
+      secrets:
+        - "my-extra-config-bp"
+  # You can still have properties here, they will be applied first
+  brokerProperties:
+    - "journal-sync-transactional=true"
+```
+
+### Practical Examples from Tests
+
+The following examples are drawn from the operator's test suite and demonstrate common configuration scenarios.
+
+#### Example 1: Basic Global Configuration
+
+This is the simplest case, where properties are applied to all broker pods in the cluster. This example sets the maximum disk usage and the minimum free disk space required.
+
+```yaml
+apiVersion: broker.amq.io/v1beta1
+kind: ActiveMQArtemis
+metadata:
+  name: ex-aao
+spec:
+  brokerProperties:
+    - "maxDiskUsage=97"
+    - "minDiskFree=5"
+```
+
+#### Example 2: Ordinal-Specific Configuration
+
+You can apply different settings to individual broker pods using an ordinal prefix (`broker-N.`). This is useful for configurations that must be unique to each broker, such as management addresses.
+
+In this example, `broker-0` and `broker-999` are assigned different disk usage limits.
```

Comment:
broker 0 and broker 1, if anyone had deploymentSize=999 it would be worrying :-)